README.md

```diff
@@ -1,4 +1,4 @@
-# MessagePack for JavaScript/ECMA-262  <!-- omit in toc -->
+# MessagePack for JavaScript/ECMA-262 <!-- omit in toc -->
```

Comment:
FYI Some prettier formatting changes in this file along with some real changes too